Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -415,8 +410,6 @@ static void OnDisconnected(NetworkConnection conn)
             }
 
             if (LogFilter.Debug) Debug.Log("Server lost client:" + conn.connectionId);
-            conn.RemoveObservers();
```

Comment:
why is conn.RemoveObservers not needed?

Reply:
Look at conn.Disconnect, it also does that